images/router/haproxy/conf/haproxy-config.template

```diff
@@ -199,7 +204,9 @@ frontend public
 # determined by the next backend in the chain which may be an app backend (passthrough termination) or a backend
 # that terminates encryption in this router (edge)
 frontend public_ssl
-  bind :::{{env "ROUTER_SERVICE_HTTPS_PORT" "443"}} v4v6 {{ if matchPattern "true|TRUE" (env "ROUTER_USE_PROXY_PROTOCOL" "") }} accept-proxy{{ end }}
+  bind :::{{env "ROUTER_SERVICE_HTTPS_PORT" "443"}}
+    {{- if matchPattern "true|TRUE" (env "ROUTER_ALLOW_IPV6" "")}} v4v6 {{ end }}
```

Comment:
Whoops!  Thanks.  I had them both this way before and then realized there was a problem (we needed to lose the :: as well).  Then forgot to change both places.